src/api/index.js

```diff
@@ -277,20 +277,21 @@ export function fetchConverstionResponse(payload) {
 
 // Botbuilder API
 
-export function fetchChatBots(payload) {
+export function fetchChatBots() {
   const url = `${API_URL}/${CMS_API_PREFIX}/getSkillList.json`;
   return ajax.get(url, {
     private: 1,
   });
 }
 
 export function fetchBotImages(payload) {
-  const { skill, group, language } = payload;
-  const url = `${API_URL}/${CMS_API_PREFIX}/getSkillList.json`;
+  const { name, language, group } = payload;
```

Comment:
skill instead of name

Reply:
@fragm3 in the getBotImages() function in BotBuilder.js the we set variable name as  bot.name, so I am passing it as name in the apis.fetchBotImages function.